components/ProjectDetails.js

```diff
@@ -15,6 +20,7 @@ export default function ProjectDetails({ currentProject, onDeleteProject }) {
     complexity,
     steps,
     image,
+    favorite,
```

Comment:
```suggestion
```
It seems like you don't use the favorite prop below, or?

Reply:
Code is part of a different US and will be fixed asap